modules/Categories/Categories.tsx

```diff
@@ -8,9 +9,9 @@ type Props = {
 const Categories = ({ categories }: Props) => (
```

Comment:
I know that it was already here, but let's update it

```suggestion
const Categories: FunctionComponent<Props> = ({ categories }) => (
```